src/mumble/MainWindow.cpp

```diff
@@ -742,16 +743,13 @@ bool MainWindow::handleSpecialContextMenu(const QUrl &url, const QPoint &pos_, b
 	return true;
 }
 
-void MainWindow::on_qtvUsers_customContextMenuRequested(const QPoint &mpos, bool usePositionForGettingContext) {
+void MainWindow::on_qtvUsers_customContextMenuRequested(const QPoint &mpos) {
```

Comment:
Why did you remove the `usePositionForGettingContext` parameter? This one is actually required in order for the context menu to work properly when invoked from the new TalkingUI.

This applies to all changes made to this function :eyes:

Reply:
I didn't actually make these changes. I suspect it is simply because my branch is not up to date with master. For now, I manually fixed these lines. What is the recommended way to git pull origin master DURING a pull request? Every time I have to make changes, I undo my last commit in order to avoid having extra commits, but then I obviously have local changes, and can't git pull until I do something with those changes. If I git pull after committing my changes, then my local branch is up to date, but to update the remote branch, I have to make another commit, which I think is not desired? What should I do in this case? I'm not particularly adept at git commands.

Reply:
To save all uncommited changes, use `git stash` before pulling.
To not introduce spurious merge commits in your PR, use auto-rebasing: https://coderwall.com/p/tnoiug/rebase-by-default-when-doing-git-pull
And then finally after you have updated your branch, use `git stash apply` to restore your uncommited changes